api.md

```diff
@@ -1036,6 +1037,60 @@ CALL distributed_exec('CREATE DATABASE dist_database', transactional => FALSE);
 
 ---
 
+## create_distributed_restore_point() [](create_distributed_restore_point)
+
+Create a named point for performing restore across a distributed database.
+
+This function is an analogue of [`pg_create_restore_point`](https://www.postgresql.org/docs/current/functions-admin.html) designed
+to work with a distributed database.
+
+On successful execution the same named restore point will be created on the access node and each
+associated data node. It synchronizes execution of ongoing remote transactions to ensure that
+each node will have a consistent restore point view across the cluster.
```

Comment:
Add something why you want this?  e.g.,:

"With such a restore point, and backup/restore point should return to the same restore point across nodes in a multi-node TimescaleDB cluster, to ensure a consistent point in time for recovery."